### PR TITLE
Explore by Region section

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds in functionality for #474 and #475 by adding an Explore by Region section to the homepage, which includes a dropdown of regions and a static world map image. Users can select a region and click to view results of that region.

Also externalizes some of the query params that are used to filter the results.

### CODE REVIEW

**Comment by mjoyce91** [comment]
@burgwyn have we updated anything with pa11y on circleci? Seems like we're getting a false positive.

**Comment by mjoyce91** [comment]
🎉 For some reason `.find` was breaking pa11y, so I instead used `.filter` and `[0]`. @burgwyn or @jseppi - are either of you aware of any cross-browser compatibility issues with `find`?

**Comment by burgwyn** [comment]
Looks like find and findIndex are unavailable in IE https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "redux-saga": "^0.15.4",
     "redux-thunk": "^2.2.0",
     "shortid": "^2.2.8",
+    "typeface-vollkorn": "0.0.35",
```
Comment (burgwyn) [inline package.json]: @shanzasubz - are we working with Volkhorn?

File: src/Components/ExploreRegionDropdown/ExploreRegionDropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  searchRegion(e) {
+    e.preventDefault();
+    this.props.onRegionSubmit(this.state.selection.value);
+  }
+
+  render() {
+    const { filters } = this.props;
+    // function to find the Region filters
+    function filterRegion(filterItem) {
```
Comment (burgwyn) [inline src/Components/ExploreRegionDropdown/ExploreRegionDropdown.jsx]: Let's pull all this logic out of `render` and into a private method

File: src/Components/FieldSet/FieldSet.jsx [hunk 3]
```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const FieldSet = ({ legend, children }) => (
+const FieldSet = ({ legend, children, legendSrOnly }) => (
   <fieldset>
-    <legend>{legend}</legend>
+    <legend className={legendSrOnly ? 'usa-sr-only' : null}>{legend}</legend>
```
Comment (burgwyn) [inline src/Components/FieldSet/FieldSet.jsx]: 👍

File: src/Constants/EndpointParams.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const ENDPOINT_PARAMS = {
```
Comment (burgwyn) [inline src/Constants/EndpointParams.js]: Are these for all endpoints or just positions?

File: src/sass/_variables.scss [hunk 5]
```diff
@@ -8,6 +8,7 @@ $color-white: #fff;
 $color-black: #000;
 $color-gray: #5b616b;
 $tm-dark-blue: #00293f;
+$tm-white-smoke: #f5f5f5;
```
Comment (burgwyn) [inline src/sass/_variables.scss]: 💨

File: src/Constants/EndpointParams.js (hunk 4 above)
Comment (mjoyce91) [reply src/Constants/EndpointParams.js]: Just positions.

File: package.json (hunk 1 above)
Comment (mjoyce91) [reply package.json]: screen shot 2017-09-08 at 8 47 36 am
screen shot 2017-09-08 at 8 47 43 am

**Comment by mjoyce91** [comment]
Issues with circle, will rebuild later.

https://status.circleci.com/

**Comment by jseppi** [comment]
Hrm, regarding the missing `.find` method in IE. We might want to include Babel's polyfill (https://github.com/babel/babel-preset-env#usebuiltins). What do you all think? If this is the only thing missing that we want, then it's probably not worth it, but if more things come up it might be a good idea.

**Pull request merged**